Failed to build proton on local MacOS

<!-- You have to provide the following information whenever possible. -->

**Describe what's wrong**
<!-- A clear and concise description of what works not as it is supposed to. -->
```
/Users/lisen/myspace/develop/proton-opensource/src/IO/Operators.h:44:70: error: no matching function for call to 'writeText'
inline WriteBuffer & operator<<(WriteBuffer & buf, const auto & x) { writeText(x, buf); return buf; }
                                                                     ^~~~~~~~~
/Users/lisen/myspace/develop/proton-opensource/src/Common/StackTrace.cpp:391:26: note: in instantiation of function template specialization 'DB::operator<<<const void *>' requested here
        out << i << ". " << addr;
                         ^
/Users/lisen/myspace/develop/proton-opensource/src/IO/WriteHelpers.h:884:13: note: candidate function not viable: cannot convert argument of incomplete type 'const void *const' to 'std::string_view' (aka 'basic_string_view<char>') for 1st argument
inline void writeText(std::string_view x, WriteBuffer & buf) { writeString(x.data(), x.size(), buf); }
            ^
/Users/lisen/myspace/develop/proton-opensource/src/IO/WriteHelpers.h:886:13: note: candidate function not viable: cannot convert argument of incomplete type 'const void *const' to 'const DayNum' (aka 'const StrongTypedef<unsigned short, DayNumTag>') for 1st argument
inline void writeText(const DayNum & x, WriteBuffer & buf) { writeDateText(LocalDate(x), buf); }
            ^
/Users/lisen/myspace/develop/proton-opensource/src/IO/WriteHelpers.h:887:13: note: candidate function not viable: cannot convert argument of incomplete type 'const void *const' to 'const LocalDate' for 1st argument
inline void writeText(const LocalDate & x, WriteBuffer & buf) { writeDateText(x, buf); }
            ^
/Users/lisen/myspace/develop/proton-opensource/src/IO/WriteHelpers.h:888:13: note: candidate function not viable: cannot convert argument of incomplete type 'const void *const' to 'const LocalDateTime' for 1st argument
inline void writeText(const LocalDateTime & x, WriteBuffer & buf) { writeDateTimeText(x, buf); }
            ^
/Users/lisen/myspace/develop/proton-opensource/src/IO/WriteHelpers.h:889:13: note: candidate function not viable: cannot convert argument of incomplete type 'const void *const' to 'const UUID' (aka 'const StrongTypedef<integer<128, unsigned int>, DB::UUIDTag>') for 1st argument
inline void writeText(const UUID & x, WriteBuffer & buf) { writeUUIDText(x, buf); }
            ^
/Users/lisen/myspace/develop/proton-opensource/src/IO/WriteHelpers.h:890:13: note: candidate function not viable: cannot convert argument of incomplete type 'const void *const' to 'const IPv4' (aka 'const StrongTypedef<unsigned int, DB::IPv4Tag>') for 1st argument
inline void writeText(const IPv4 & x, WriteBuffer & buf) { writeIPv4Text(x, buf); }
            ^
/Users/lisen/myspace/develop/proton-opensource/src/IO/WriteHelpers.h:891:13: note: candidate function not viable: cannot convert argument of incomplete type 'const void *const' to 'const IPv6' for 1st argument
inline void writeText(const IPv6 & x, WriteBuffer & buf) { writeIPv6Text(x, buf); }
            ^
```
**How to reproduce**

**Error message and/or stacktrace**
<!-- If applicable, add screenshots to help explain your problem. -->

**Additional context**
<!--Add any other context about the problem here. -->

--- a/src/Common/StackTrace.cpp
+++ b/src/Common/StackTrace.cpp
@@ -387,10 +387,7 @@ static void toStringEveryLineImpl(
     for (size_t i = offset; i < size; ++i)
     {
         const void * addr = frame_pointers[i];
-        DB::WriteBufferFromOwnString out;
-        out << i << ". " << addr;
-
-        callback(out.str());
+        callback(fmt::format("{}. {}", i, addr));
     }
 #endif
 }

